cuda_core/examples/strided_memory_view.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+# ################################################################################
+#
+# This demo aims to illustrate two takeaways:
+#
+#   1. The similarity between CPU and GPU JIT-compilation for C++ sources
+#   2. How to use StridedMemoryView to interface with foreign CPU/GPU functions
+#      at low-level
+#
+# To facilitate this demo, we use cffi (https://cffi.readthedocs.io/) for the CPU
+# path, which can be easily installed from pip or conda following their instruction.
+# We also use NumPy/CuPy as the CPU/GPU array container.
+#
+# ################################################################################
+
+import string
+import sys
+
+try:
+    from cffi import FFI
+except ImportError:
+    print("cffi is not installed, the CPU example would be skipped", file=sys.stderr)
+    cffi = None
```

Comment:
FFI = None

I discovered that because I didn't have cffi when running the tests.

Which brings me to another question: Would it make sense to add cffi to the test dependencies (e.g. `extras_require` in setup.py)?